Sources/Sentry/Public/SentryClient.h

```diff
@@ -95,9 +96,19 @@ SENTRY_NO_INIT
 /**
  * Captures a manually created user feedback and sends it to Sentry.
  * @param userFeedback The user feedback to send to Sentry.
+ * @deprecated Use @c -[SentryClient @c captureFeedback:withScope:] .
  */
 - (void)captureUserFeedback:(SentryUserFeedback *)userFeedback
-    NS_SWIFT_NAME(capture(userFeedback:));
+    NS_SWIFT_NAME(capture(userFeedback:))
+        DEPRECATED_MSG_ATTRIBUTE("Use -[SentryClient captureFeedback:withScope:].");
+
+/**
+ * Captures a new-style user feedback and sends it to Sentry.
```

Comment:
`m`: Again maybe we should talk about V2 instead of new-style, cause this is a public API.